src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
```

Comment:
Consider using a separate method in FigureSupplier to generate random values for the radius and pass those to the constructor for better code readability, instead of using expressions directly in the constructor call.